src/main.rs

```diff
@@ -8,9 +8,14 @@ use aptos_types::{
 use chrono::prelude::{DateTime, Local};
 use clap::Parser;
 use crossbeam::channel;
+use once_cell::sync::Lazy;
+use regex::Regex;
 use std::thread::{self, available_parallelism};
 use std::time::{Instant, SystemTime};
 
+/// Regex for checking that prefix only includes valid hex characters.
+static HEX_REGEX: Lazy<Regex> = Lazy::new(|| Regex::new(r"^[0-9a-fA-F]+$").unwrap());
```

Comment:
seems like a lot of effort importing libraries to make this static. Why not make it run in-line since it only runs once anyway?